src/main/java/frc/team2412/robot/util/SwerveConfiguration.kt

```diff
@@ -0,0 +1,10 @@
+package frc.team2412.robot.util
+
+import com.swervedrivespecialties.swervelib.Mk4SwerveModuleHelper
+import com.swervedrivespecialties.swervelib.SwerveModule
+//kotlin is very fun
+data class Mk4Configuration(val ratio: Mk4SwerveModuleHelper.GearRatio, val drive: Int, val angle: Int, val encoder: Int, val offset: Double){
+    fun falcons(): SwerveModule {
```

Comment:
Please rename the file to match the class name, so it's easier for developers to find.